opal/corelib/error.rb

```diff
@@ -224,7 +224,6 @@ class ::ZeroDivisionError   < ::StandardError; end
 class ::NameError           < ::StandardError; end
 class ::NoMethodError         < ::NameError; end
 class ::RuntimeError        < ::StandardError; end
-class ::FrozenError           < ::RuntimeError; end
```

Comment:
This shouldn't be done here, the error tree is of an importance for maintaining.